test/preprocessing/test_preprocessing.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+import cv2
+from pathlib import Path
+import numpy as np
+from fruit_classifier.preprocessing.preprocessing_utils import preprocess_image
+
+
+class TestPreprocessingUtils(unittest.TestCase):
+
+    def setUp(self):
+
+        test_dir = Path(__file__).absolute().parents[1]
+        self.jpg_image_file_name = test_dir.joinpath("test_data",
+                                                     "original_test_image.jpg")
+        self.png_image_file_name = test_dir.joinpath("test_data",
+                                                     "original_test_image.png")
+
+        self.test_orig_shape = [115, 73, 3]
+        self.test_orig_max = 255
+        self.test_orig_min = 0
+
+        self.test_comp_shape = [28, 28, 3]
+        self.test_comp_max = 0.00392156862745098
```

Comment:
These are very precise numbers that may depend on hardware. We would like to have less stringent tests